### PR TITLE
Add page Home

### PR DESCRIPTION
**Pull request by teifler**
Thanks for reviewing that is a large pull request for the first user story

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+          try it again!
+        </ErrorMessage>
+      )}
+    </>
   );
 }
 
 export default App;
+
+const ErrorMessage = styled.h2`
```
Comment (FaGru) [inline src/App.js:48]: h2? maybe a normal `<p> `is enough?

File: src/Home.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                key={coin.id}
+                name={coin.name}
+                symbol={coin.symbol}
+                image={coin.image}
+                price={coin.current_price}
+                price_change_percentage_24h={coin.price_change_percentage_24h}
+              />
+            );
+          })}
+          {filterdCoins == [] ? (
```
Comment (FaGru) [inline src/Home.js:68]: maybe `filterdCoins.length === 0 ?` is better?

**Review by FaGru — APPROVED** [review]
Looks good to me. Maybe 2 little changes if you want.

File: src/Home.js (hunk 2 above)
Comment (teifler) [reply src/Home.js:68]: Thanks for checking mate! I changed the code and added the two improvements.

File: src/components/CoinCard.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+          </ColorRed>
+        )}
+      </CoinDetails>
+    </CardWrapper>
+  );
+}
+
+export default CoinCard;
+
+const CardWrapper = styled.div`
```
Comment (GT458) [inline src/components/CoinCard.js:60]: Same comment as above, maybe in their own file?

**Review by GT458 — APPROVED** [review]
Left a few notes about separating styling from your component files, but otherwise LGRM

File: src/components/Header.spec.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import Header from './Header.js';
+
+describe('Header', () => {
+  it('renders h1 with pagetitel', () => {
```
Comment (FNKMYR) [inline src/components/Header.spec.js:5]: `pagetitel`
Let's be a little nitpicky

**Review by FNKMYR — APPROVED** [review]

File: src/components/CoinCard.js (hunk 3 above)
Comment (teifler) [reply src/components/CoinCard.js:60]: Hey I did the styling with Styled components so that I can do in the same file :)

File: src/Home.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+    if (e.target.value.trim() !== '') {
+      if (!e.target.value.trim().match(/^[a-zA-Z0-9_.]+$/)) {
+        setSearchError(true);
+      } else {
+        setSearchError(false);
+      }
+    } else {
+      setSearchError(false);
+    }
```
Comment (MichiBerger) [inline src/Home.js:26]: good work so far 👍  do you really need to nest the second div?

**Pull request merged**